pmm/pmm2-ovf-test.groovy

```diff
@@ -0,0 +1,212 @@
+pipeline {
+    agent {
+        label 'virtualbox'
+    }
+    parameters {
+        string(
+            defaultValue: 'dev-latest',
+            description: 'PMM Client version',
```

Comment:
PMM2 ?

Reply:
I think - yes, but better  clarify this  in the description